docs/deployment/advanced-database-migrations.mdx

```diff
@@ -23,129 +28,57 @@ Alternatively, you can add the script to your `PATH` and run it from anywhere wi
 export PATH="$HOME/path/to/deployment:$PATH"
```

Comment:
This comment belongs on **line 22** above, I can't put it there because Github...

This command `python aerie_db_migration.py` is no longer a valid call to the script due to the newly added subcommands - we should replace it with the new minimum command. Since I think some values from the env file are required, that would be something like  `aerie_db_migration.py migrate -e .env`, correct?

Reply:
`-e .env` isn't required, but the subcommand is. I've updated the docs to reflect this.